### DISCUSSION TITLE
Stay signed in after an offline session

### DISCUSSION THREAD

**Discussion by ravener** [ppy/osu#27863] [Ideas] [answered]
In stable I can occasionally play offline, open the game offline, do stuff, and next time I reconnect I can still re submit all scores of that session (i.e provided the game wasn't closed), all smooth.

In Lazer, after an offline session, I'm completely logged off and next time I reconnect I have to re-login.

While offline score submission was already discussed in #19912 there at least needs to be a way for an easier reconnecting, because I'm forced to enter my password again and go through email 2fa verification code to log back in.

Either 'stay signed in' should really keep us signed in, just like in stable, or at the very least the 2FA could be relaxed, such as if we are logging in from the same IP or device to make re-logins less of an issue.

**Comment by peppy** [comment, marked as answer]
A network connection is required to play. This will not change.
